Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor.yml

```diff
@@ -0,0 +1,880 @@
+        after the specified date/time will be retrieved. If no time zone is specified,
+        UTC time zone will be used.
+      name: start_time
+    - description: The end date in ISO date format, epoch time or time range(<number>
+        <time unit>', e.g., 12 hours, 7 days). Only alerts that were created on or
+        before the specified date/time will be retrieved. If no time zone is specified,
+        UTC time zone will be used.
+      name: end_time
+    - auto: PREDEFINED
+      description: If true, retrieves alerts that have been resolved. If false, retrieves
+        alerts that have not been resolved.
+      name: resolved
+      predefined:
+      - "true"
+      - "false"
+    - auto: PREDEFINED
+      description: If true, retrieves alerts that have been resolved, and were auto_resolved.
+        If false, retrieves alerts that have been resolved, and were not auto_resolved.
+      name: auto_resolved
+      predefined:
+      - "true"
+      - "false"
+    - auto: PREDEFINED
+      description: If true, retrieves alerts that have been acknowledged. If false,
+        retrieves alerts that have not been acknowledged.
+      name: acknowledged
+      predefined:
+      - "true"
+      - "false"
+    - auto: PREDEFINED
+      description: Comma-separated list of the severity levels of the alerts to retrieve.
+      name: severity
+      predefined:
+      - CRITICAL
+      - WARNING
+      - INFO
+      - AUDIT
+    - description: Comma-separated list of alert type IDs. Will retrieve alerts whose
+        type ID matches an alert_type_id in the alert_type_ids list. For example,
+        alert 'Alert E-mail Failure' has type id of A111066. Given alert_type_ids=
+        'A111066', only alerts of 'Alert E-mail Failure' will be retrieved.
+      isArray: true
+      name: alert_type_ids
+    - auto: PREDEFINED
+      description: "Comma-separated list of impact types. Possible values: Availability,\
+        \ Capacity, Configuration, Performance, and System Indicator. Will retrieve\
+        \ alerts whose impact type matches an impact types in the impact_types list.\
+        \ For example, alert 'Incorrect NTP Configuration' has impact type 'SystemIndicator'.\
+        \ Given impact_types ='SystemIndicator',only alerts with impact type 'SystemIndicator',\
+        \ such as 'Incorrect NTP Configuration' will be retrieved."
+      isArray: true
+      name: impact_types
+      predefined:
+      - Availability
+      - Capacity
+      - Configuration
+      - Performance
+      - System Indicator
+    - description: Comma-separated list of entity types. Will retrieve alerts whose
+        entity_type matches an entity_type in the entity_types list. For details,
+        see the Nutanix README. If the Nutanix service cannot recognize the entity
+        type, it returns a 404 error.
+      isArray: true
+      name: entity_types
+    - description: Page number in the query response. Default is 1.
```

Comment:
if the default is 1 then set it as `defaultValue` and remove it from the description as the platform adds it automatically